Manual.md

```diff
@@ -963,7 +970,9 @@ additional paths to be searched when linking target binaries to be introspected.
 - `qemu` sets additional variables for the `cmake` and `meson` build styles to allow
 executing cross-compiled binaries inside qemu.
 It sets `CMAKE_CROSSCOMPILING_EMULATOR` for cmake and `exe_wrapper` for meson
-to `qemu-<target_arch>-static` and `QEMU_LD_PREFIX` to `XBPS_CROSS_BASE`
+to `qemu-<target_arch>-static` and `QEMU_LD_PREFIX` to `XBPS_CROSS_BASE`.
+It also creates the `vtargetrun` function to call the target architecture
```

Comment:
Nitpicks: the target architecture -> the target architecture's?

Reply:
If we're going to reword, "[...] function to call the `qemu-<arch>-static` executable for the target architecture." Possessive inanimate objects are weird.

Or maybe better, "[...] function to wrap commands in a call to `qemu-<arch>-static` for the the target architecture."